Tests/NIOTests/SelectorTest.swift

```diff
@@ -96,5 +97,257 @@ class SelectorTest: XCTestCase {
         }
         XCTAssertEqual(1, readyCount)
     }
-}
 
+    private static let testWeDoNotDeliverEventsForPreviouslyClosedChannels_numberOfChannelsToUse = 10
+    func testWeDoNotDeliverEventsForPreviouslyClosedChannels() {
+        /// We use this class to box mutable values, generally in this test anything boxed should only be read/written
+        /// on the event loop `el`.
```

Comment:
@weissi should we use a computed property and check `MultiThreadedEventLoopThread.currentEventLoop` returns what we expect ?

Reply:
sure why not